modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -5,16 +5,17 @@ endif::[]
 :version-info: 2.0 and newer
 :keywords: munit, testing, unit testing
 :page-aliases: munit-maven-support.adoc, to-set-up-munit-maven-plugin.adoc, faq-working-with-parent-pom.adoc, to-inherit-parent-plugin.adoc, to-ignore-parent-plugin.adoc, to-override-parent-plugin.adoc, munit-maven-plugin-configuration.adoc,to-configure-munit-maven-plugin-maven.adoc
+:page-deployment-options: acb-ide, studio-ide
```

Comment:
I don't see the pills in my local build

Reply:
It seems the local build is missing some features. 
The pills look great on your beta site: https://dev-docs-internal.kqa.msap.io/beta-code-builder/munit/latest/
Thanks!